config/core/test-configs.yaml

```diff
@@ -1635,6 +1635,23 @@ device_types:
       - blocklist: *device_config_filter
       - regex: {defconfig: '^defconfig'}
 
+  qemu_smp8_riscv64:
```

Comment:
How about using a YAML anchor to avoid duplicating all the common parts?  The only thing that differs afaict is the `extra_options`.

Reply:
Instead of creating a new device_type for smp=8, please just make the current device type default to smp=8, which is what we want for all the existing tests

As follow-on work, we need to explore how to update/modify QEMU cmdline args based on kernel options (e.g. defconfig.)  For example if the defconfig has `CONFIG_SMP=n` then we should not have any `-smp` on the cmdline.

Reply:
After discussion with @montjoie we decided that having the existing no-smp device and smp8 would both be useful.

Reply:
@gctucker FYI: an issue was created for this back in Dec: https://github.com/kernelci/kernelci-core/issues/1579

Reply:
Tip: Adding "Fixes #1579" in the PR description helps in this respect.  The only gotcha is that the issue will now be closed automatically when this PR is merged, which is desirable as long as it really solves the issue entirely.